src/main/java/com/google/ehub/servlets/ItemPageServlet.java

```diff
@@ -88,4 +88,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       }
     }
   }
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+    long commentId;
+    CommentDataManager comment = new CommentDataManager();
+    try {
+      commentId = Long.parseLong(request.getParameter("commentId"));
+    } catch (NumberFormatException e) {
+      System.err.println("Can't parse itemId to a long");
```

Comment:
prefer to include the exception in the print statement, which can have more details and make it easier to debug.